mypy/nodes.py

```diff
@@ -1518,18 +1520,30 @@ def accept(self, visitor: ExpressionVisitor[T]) -> T:
 
 # Kinds of arguments
 
-# Positional argument
-ARG_POS: Final = 0
-# Positional, optional argument (functions only, not calls)
-ARG_OPT: Final = 1
-# *arg argument
-ARG_STAR: Final = 2
-# Keyword argument x=y in call, or keyword-only function arg
-ARG_NAMED: Final = 3
-# **arg argument
-ARG_STAR2: Final = 4
-# In an argument list, keyword-only and also optional
-ARG_NAMED_OPT: Final = 5
+class ArgKind(Enum):
+    # Positional argument
+    ARG_POS = 0
+    # Positional, optional argument (functions only, not calls)
+    ARG_OPT = 1
+    # *arg argument
+    ARG_STAR = 2
+    # Keyword argument x=y in call, or keyword-only function arg
+    ARG_NAMED = 3
+    # **arg argument
+    ARG_STAR2 = 4
+    # In an argument list, keyword-only and also optional
+    ARG_NAMED_OPT = 5
+
+    def is_normal(self) -> bool:
```

Comment:
"normal" is pretty ambiguous. Perhaps `is_single_arg`, or `is_starred` for the reverse method.